language/bytecode-verifier/bytecode-verifier-tests/src/unit_tests/bounds_tests.rs

```diff
@@ -157,6 +192,23 @@ fn invalid_struct_as_type_actual_in_exists() {
     m.freeze().unwrap_err();
 }
 
+#[test]
+fn script_invalid_struct_as_type_actual_in_exists() {
```

Comment:
FYI: we had a historical debate on terminologies: `(type) formal + (type) actual` vs `(type) parameter + (type) argument`. We didn't reach a conclusion immediately, but over time, at least from my view, the latter became the de facto winner. It would be better if we would keep our usages consistent.

Reply:
Oh sorry for the confusion, I meant renaming `script_invalid_struct_as_type_actual_in_exists` to `script_invalid_struct_as_type_argument_in_exists`. 

Let's keep the script prefix since it's more explicit.